### PR TITLE
Add igwn-cmake

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for `igwn-cmake` a cmake modules library.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@emaros, please post here if you are happy to be named as co-maintainer.

**Comment by emaros** [comment]
> @emaros, please post here if you are happy to be named as co-maintainer.

I am willing to be co-maintainer.

**Comment by duncanmmacleod** [comment]
@conda-forge/stages-recipes, I think this recipe is ready for review.

This package is `noarch: generic` so I presume that means in production it will just include builds on `linux` is that right? I haven't provided a windows build script for that reason.

**Comment by emaros** [comment]
> @conda-forge/stages-recipes, I think this recipe is ready for review.
> 
> This package is `noarch: generic` so I presume that means in production it will just include builds on `linux` is that right? I haven't provided a windows build script for that reason.

That is not correct. It will be providing a collection of CMake macros/functions in the form of .cmake files and some pure python scripts. If possible, it would be good to have recipes for windows.

**Comment by scopatz** [comment]
noarch: generic will only build on the linux machines and then apply the same filesystem layout to all platforms on install.  In this recipe, it is probably a good idea to build on each platform

File: recipes/igwn-cmake/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  url: http://software.igwn.org/sources/source/{{ name }}-{{ version }}.tar.gz
+  sha256: c86ac84e53cbb141aaa58087de5e442bdb122db5af4c119dc7c89129f261da21
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - pkg-config
```
Comment (scopatz) [inline recipes/igwn-cmake/meta.yaml:18]: I think you might need make here on unix, because it is used implicitly

Comment (duncanmmacleod) [reply recipes/igwn-cmake/meta.yaml:19]: Done in 952e12b, thanks @scopatz.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**